gradle.properties

```diff
@@ -0,0 +1,3 @@
+org.gradle.java.installations.auto-download=false
```

Comment:
Turned off auto download and setup environmental variables for the jdk versions (I wasn't sure /opt/java would be auto discovered).